.husky/pre-commit

```diff
@@ -3,3 +3,4 @@
 
 npm run format
 npm run test
+git add .
```

Comment:
I was having issues with the lint-staged formatting not being committed therefore failing the code format checks. is there a better way of doing this?

Reply:
Yes I've noticed this also. First commit fails because of lint errors so I just add the newly linted files again and then commit again. Possibly you could run the linting command manually before committing.